solution/backend/regulations/views/policy_repository.py

```diff
@@ -12,7 +12,7 @@ def get_context_data(self, **kwargs):
         host = self.request.get_host()
         is_authenticated = self.request.user.is_authenticated
         user_groups = [group.name for group in self.request.user.groups.all()]
-        has_editable_job_code = user_groups and user_groups != ['EREGS-READER']
+        has_editable_job_code = user_groups and user_groups != ['EREGS_READER']
```

Comment:
Probably not good logic here.  What if they had a user group that is not eregs related and only eregs reader.  They would then by definition have an editable job code.  I would make the logic to check that they are in a group that can edit, not one that cannot.

Reply:
Why would they have a user group that is not EREGS? These are user groups not jobcodes. We only offer eregs user groups. None eregs job codes will not get assigned to a user group?